### DISCUSSION TITLE
龙de船人 模板问题反馈评论区

### DISCUSSION THREAD

**Discussion by wjf0214** [wjf0214/qd-templates#74] [模板讨论]
龙de船人 模板问题反馈评论区
有问题可以下面反馈。

**Comment by miaoxiaozhao** [comment]
Failed at 2/6 request,
Fail assert: {"re": "200", "from": "status"} from success_asserts,
Response Error : HTTP 403: Forbidden,
Request URL: https://club.imarine.cn/qiandao/
